runtime/vara/src/lib.rs

```diff
@@ -136,13 +138,23 @@ pub const BABE_GENESIS_EPOCH_CONFIG: sp_consensus_babe::BabeEpochConfiguration =
         allowed_slots: sp_consensus_babe::AllowedSlots::PrimaryAndSecondaryPlainSlots,
     };
 
+// We'll check that WEIGHT_REF_TIME_PER_SECOND is not overflows so we can use checked_mul and
+// checked_div instead of saturating ones.
+const_assert!(WEIGHT_REF_TIME_PER_SECOND
+    .checked_div(3)
+    .expect("cannot divide WEIGHT_REF_TIME_PER_SECOND by 3")
```

Comment:
We can avoid `expect` here if replacing this `const_assert` with two. So, we can omit the `const_option` feature.

Reply:
Thanks! Done